### DISCUSSION TITLE
When a version compatible with the oqtane 5.1.0  will be released

### DISCUSSION THREAD

**Discussion by niknam-mohsen** [2sic/2sxc#3353] [Q&A]
The latest version of 2sxc (17.06.03 LTS) is not compatible with oqtane 5.1.0. When will the compatible version be released?

**Comment by iJungleboy** [comment]
2sxc already has a branch which works, but we're holding back. 

The reason we just published here https://docs.2sxc.org/abyss/platforms/oqtane/install.html

Background is that Oqtane 5.1 is still fairly unstable from our point of view, and releasing a 2sxc would result in a lot of support / tracking down problems which are not caused by 2sxc.

EG ATM sub-sites in Oqtane cause a lot of problems etc. (should be fixed in 5.1.2) and some SSR Modes (auto) still don't really work at all. 

We hope that 5.1.2 or so should be good, and then we'll release.

**Comment by iJungleboy** [comment]
Just an update on this.

We've been working hard on 5.1.2 - basically everything that worked before, broke. So I guess it was a good decision to wait.

But unfortunately we see 5.1.2 as surprisingly unstable as well. 

I assume we just see all the problems because we're testing all kinds of use cases, and at least these are all still kind of unstable:

1. Sub sites with path, eg. main site `localhost:1234/` and sub site `localhost:1234/products` break out of the box without 2sxc
2. various render modes fail even on the main site

Within these conditions, we somehow are forced to patch around all kinds of bugs, as users will quickly run into problems. So we're still forced to wait.
